common/src/main/scala/explore/model/RootModel.scala

```diff
@@ -14,6 +14,7 @@ import monocle.macros.Lenses
 import cats.effect.concurrent.Ref
 import japgolly.scalajs.react._
 import diode.data._
+import explore.util.Pot._
```

Comment:
Did you internalize `Pot`?

Reply:
Not yet. For the moment in that object we have some utils for working with it.